telethon/client/updates.py

```diff
@@ -459,10 +459,10 @@ def _process_update(self: 'TelegramClient', update, others, entities=None):
 
             if not pts_date:
                 # First-time, can't get difference. Get pts instead.
-                result = await self(functions.messages.GetPeerDialogsRequest([
-                    utils.get_input_dialog(where)
-                ]))
-                self._state_cache[channel_id] = result.dialogs[0].pts
+                result = await self(functions.channels.GetFullChannelRequest(
+                    channel=where
```

Comment:
For symmetry with the previous changes…
```suggestion
                    utils.get_input_channel(where)
```